php/driftphp/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "drift/skeleton",
+    "description": "Skeleton repository ofr DriftPHP",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Marc Morera",
+            "email": "yuhu@mmoreram.com"
+        }
+    ],
+    "require": {
+        "php": "^7.3",
+        "symfony/console": "^5.0.0",
```

Comment:
already required in https://github.com/driftphp/server/blob/c36d51be73afd36bc9c6d7bd1070ce1676a698b3/composer.json#L17

Reply:
I remove this dependency because, in this case, the console is not used directly. This dependency must be present in the skeleton, as the console is used directly there, outside the server.